src/lib/types/stores/store.ts

```diff
@@ -1,5 +1,5 @@
 export interface Store<T, K> {
-    get(key: K): Promise<T>;
+    get(key: K): Promise<T | undefined>;
```

Comment:
I've been willing to make this change for a long time and it seems nothing is breaking, you might expect a few compiler warnings here and there (probably everywhere), but the truth is when you get an element by key you may not get the element if the key does not exist